nodepool/files/elements/jenkins-slave/install.d/20-jenkins-slave

```diff
@@ -20,6 +20,14 @@ set -o pipefail
 source /etc/lsb-release
 if [[ "${DISTRIB_CODENAME}" != "trusty" ]] && [[ "${DISTRIB_CODENAME}" != "xenial" ]]; then
     systemctl disable systemd-resolved
+    # Remove the systemd-resolved stub file
+    # symlink. glean will put a normal file
+    # in its place on boot.
+    rm -f /etc/resolv.conf
+    # To allow the image build to complete,
+    # we add a replacement file with a public
+    # resolver.
+    echo 'nameserver 8.8.8.8' > /etc/resolv.conf
```

Comment:
if we have glean always available we could rerun it to generate the correct resolv.conf

Reply:
sounds like glean is always available, rerun it with...

`glean` (no options should be needed)

Reply:
A config-drive isn't available during the diskimage-builder process, so glean won't work here.